### PR TITLE
Error message is no longer verbose

### PR DESCRIPTION
**Pull request by scribe**

### CODE REVIEW

File: dsb-gui/src/main/resources/lang_en.properties [hunk 1]
```diff
@@ -195,7 +194,8 @@ toggleShowCachedJobs=Toggle show Jobs "In Cache" or "Persisted"
 putJobStarted=Put Job Started
 blackPearlHealth=Checking BlackPearl's health
 at=at
-transferringEllipsis=Transferring...
+transferringEllipsis=Transferring...tFailedErr"), LogType.ERROR);
```
Comment (rpmoore) [inline dsb-gui/src/main/resources/lang_en.properties]: This looks like find and replace got a little too excited.

File: dsb-gui/src/main/resources/lang_en.properties [hunk 2]
```diff
@@ -195,7 +194,7 @@ toggleShowCachedJobs=Toggle show Jobs "In Cache" or "Persisted"
 putJobStarted=Put Job Started
 blackPearlHealth=Checking BlackPearl's health
 at=at
-transferringEllipsis=Transferring...
+transferringEllipsis=Transferring...;
```
Comment (rpmoore) [inline dsb-gui/src/main/resources/lang_en.properties]: Is the semi-colon there on purpose?

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateBucketTask.java [hunk 3]
```diff
@@ -53,9 +55,14 @@ protected Object call() throws Exception {
         try {
             return ds3Client.putBucketSpectraS3(new PutBucketSpectraS3Request(bucketName)
                     .withDataPolicyId(createBucketModel.getId()));
-        } catch (final Exception e) {
+        } catch (final FailedRequestException fRE) {
+            final Joiner joiner = Joiner.on(", ");
+            LOG.error("Failed to create bucket", fRE);
+            loggingService.logMessage("While creating bucket expected status code " + joiner.join(fRE.getExpectedStatusCodes()) + " but got " + fRE.getStatusCode(), LogType.ERROR);
```
Comment (rpmoore) [inline dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateBucketTask.java]: Is there a way we can get a portion of the error message to determine what went wrong?  Or if not the whole message, determine which one it is and display our own custom error message?

**Comment by shabtaisharon** [comment]
This branch is out-of-date with the base branch

File: dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateBucketTask.java (hunk 3 above)
Comment (scribe) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateBucketTask.java]: I agree, perhaps we should make this less verbose for now, and work on a translation between BP Errors and User errors?

Comment (rpmoore) [reply dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateBucketTask.java]: That seems like a reasonable approach.

**Comment by rpmoore** [comment]
Github doesn't seem to want to merge this.  Please update the pr locally and then re-push to update.

**Pull request merged**